bundles/debian/Containerfile

```diff
@@ -0,0 +1,51 @@
+# Copyright 2022 The cert-manager Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM docker.io/library/debian:11-slim as debbase
+
+ARG EXPECTED_VERSION
+ARG VERSION_SUFFIX
+
+WORKDIR /work
+
+COPY ./build.sh /work/build.sh
+
+RUN /work/build.sh $EXPECTED_VERSION $VERSION_SUFFIX /work/bundle.json
+
+FROM docker.io/library/golang:1.19 as gobuild
+
+WORKDIR /work
+
+COPY go.mod go.mod
+COPY go.sum go.sum
+COPY main.go main.go
+
+RUN CGO_ENABLED=0 go build -o copypause main.go
+
+FROM scratch
+
+ARG EXPECTED_VERSION
+ARG VERSION_SUFFIX
+
+LABEL description="cert-manager debian static trust bundle"
+
+USER 1001
+
+COPY --from=debbase /usr/bin/tini-static /tini
+COPY --from=debbase /work/bundle.json /packaged-bundles/bundle-debian-$EXPECTED_VERSION$VERSION_SUFFIX.json
+COPY --from=gobuild /work/copypause /copypause
+
+ENTRYPOINT ["/tini", "--"]
```

Comment:
Why do we need tini, instead of calling `copypause` directly?

Reply:
For the reasons listed in [Tini's README](https://github.com/krallin/tini#why-tini). I tend to believe that most containers should be using an init entrypoint like Tini because it'll handle zombies and signals correctly by default. Certainly none of the cert-manager / trust-manager containers reap zombies and I would love to make the change for using tini everywhere.

Obviously there's very little chance of zombie processes being created by the static bundle, but I wanted the container to be as ironclad as I could make it from the get-go.